tools/build-dev-env.js

```diff
@@ -6,6 +6,8 @@ const setEnvVariable = require('./set-env-var')
 
 fs.copyFileSync(path.resolve(__dirname, '../.sample.env'), path.resolve(__dirname, '../.env'))
 
+setEnvVariable('NODE_ENV', 'development')
```

Comment:
This already defaults to development, no?

Reply:
I'm on WSL2 and it's defaulting to undefined